<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>366</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>배경 카드번호 등록</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>68</y>
     <width>56</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>은행명 :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="card_bank_name_widget">
   <property name="geometry">
    <rect>
     <x>74</x>
     <y>65</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>222</x>
     <y>66</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>카드번호:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="card_account_no_widget">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>65</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>100</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>사용부서 :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="card_depart_widget">
   <property name="geometry">
    <rect>
     <x>88</x>
     <y>98</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QTableWidget" name="card_tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>501</width>
     <height>192</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <column>
    <property name="text">
     <string>은행명</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>계좌번호</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>사용부서</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="delete_selected_Button">
   <property name="geometry">
    <rect>
     <x>414</x>
     <y>330</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>선택행 삭제</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_card_list_Button">
   <property name="geometry">
    <rect>
     <x>314</x>
     <y>20</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>추가저장(F12)</string>
   </property>
   <property name="shortcut">
    <string>F12</string>
   </property>
  </widget>
  <widget class="QPushButton" name="close_Button">
   <property name="geometry">
    <rect>
     <x>434</x>
     <y>20</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>종료</string>
   </property>
  </widget>
  <widget class="QPushButton" name="card_view_Button">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>94</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>보기</string>
   </property>
  </widget>
  <widget class="QPushButton" name="modify_selected_Button">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>330</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>선택행 수정</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>add_card_list_Button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>carddata_append()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>371</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel">
     <x>321</x>
     <y>33</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>close_Button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>addcard_close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>473</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>443</x>
     <y>51</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>card_view_Button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>card_view()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>471</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>404</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>delete_selected_Button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>delete_selected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>445</x>
     <y>336</y>
    </hint>
    <hint type="destinationlabel">
     <x>405</x>
     <y>336</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>modify_selected_Button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>modify_selected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>293</x>
     <y>340</y>
    </hint>
    <hint type="destinationlabel">
     <x>253</x>
     <y>337</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>card_save()</slot>
  <slot>addcard_close()</slot>
  <slot>carddata_append()</slot>
  <slot>card_view()</slot>
  <slot>delete_selected()</slot>
  <slot>modify_selected()</slot>
 </slots>
</ui>
